src/main/scala/com/microsoft/partnercatalyst/fortis/spark/ProjectFortis.scala

```diff
@@ -168,6 +170,19 @@ object ProjectFortis extends App {
               "consumerGroup" -> "$Default"
             )
           )
+        ),
+        "customevents" -> List(
+          ConnectorConfig(
+            "CustomEvents",
+            Map (
+              "policyName" -> envOrElse("CUSTOMEVENTS_EH_POLICY_NAME", "project-fortis-spark"),
```

Comment:
Unfortunately the Spark connector for EventHubs [doesn't support the connection string](https://github.com/hdinsight/spark-eventhubs/blob/master/docs/direct_stream.md#how-we-use-it). I'll update the GQL library instead.